doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -535,24 +543,125 @@ fulfill plant nitrogen demands.
+ +----------------------------------+----------------------------+
+ | BDT tropical                     |             0.00           |
+ +----------------------------------+----------------------------+
+ | BDT temperate                    |             0.00           |
+ +----------------------------------+----------------------------+
+ | BDT boreal                       |             0.00           |
+ +----------------------------------+----------------------------+
+ | BES temperate                    |             0.00           |
+ +----------------------------------+----------------------------+
+ | BDS temperate                    |             0.00           |
+ +----------------------------------+----------------------------+
+ | BDS boreal                       |             0.00           |
+ +----------------------------------+----------------------------+
+ | C\ :sub:`3` arctic grass         |             0.00           |
+ +----------------------------------+----------------------------+
+ | C\ :sub:`3` grass                |             0.00           |
+ +----------------------------------+----------------------------+
+ | C\ :sub:`4` grass                |             0.00           |
+ +----------------------------------+----------------------------+
+ | Temperate Corn                   |             0.00           |
+ +----------------------------------+----------------------------+
+ | Spring Wheat                     |             0.00           |
+ +----------------------------------+----------------------------+
+ | Temperate Soybean                |             0.00           |
+ +----------------------------------+----------------------------+
+ | Cotton                           |             0.00           |
+ +----------------------------------+----------------------------+
+ | Rice                             |             0.00           |
+ +----------------------------------+----------------------------+
+ | Sugarcane                        |             0.00           |
+ +----------------------------------+----------------------------+
+ | Tropical Corn                    |             0.00           |
+ +----------------------------------+----------------------------+
+ | Tropical Soybean                 |             0.00           |
+ +----------------------------------+----------------------------+
+ | Miscanthus                       |             0.70           |
+ +----------------------------------+----------------------------+
+ | Switchgrass                      |             0.70           |
+ +----------------------------------+----------------------------+
+
+Whereas food C and N was formerly transferred to the litter pool, CLM5 routes food C and N
 to a grain product pool where the C and N decay to the atmosphere over one year,
 similar in structure to the wood product pools. 
+The biofuel C and N is also routed to the grain product pool and decay to the atmosphere over one year.
```

Comment:
The next sentence mentions seeding. Since biofuels are perennial crops, is this also used for biofuels?